examples/query-playground/package.json

```diff
@@ -5,24 +5,30 @@
     "license": "BSD-3-Clause",
     "private": true,
     "scripts": {
-        "override": "cd .d2/shell && yarn add --force @dhis2/app-runtime@file:../../../../runtime",
-        "build": "d2-app-scripts build --force",
-        "build:standalone": "d2-app-scripts build --force --standalone",
-        "start": "yarn override && d2-app-scripts start",
-        "test": "yarn override && d2-app-scripts test"
+        "build": "d2-app-scripts build",
+        "build:standalone": "d2-app-scripts build --standalone",
+        "start": "d2-app-scripts start",
+        "test": "d2-app-scripts test"
     },
     "devDependencies": {
-        "@dhis2/cli-app-scripts": "^1.5.1"
+        "@dhis2/cli-app-scripts": "^4.0.0"
     },
     "dependencies": {
-        "@dhis2/app-runtime": "^2.0.1",
-        "brace": "^0.11.1",
-        "react-ace": "^7.0.4"
-    },
-    "peerDependencies": {
+        "@dhis2/app-runtime": "file:../../runtime",
+        "@dhis2/app-service-config": "file:../../services/config",
+        "@dhis2/app-service-data": "file:../../services/data",
```

Comment:
Is it correct that the peerDeps should be linked to "file:"  when "resolutions" is used below?

Reply:
I think it's the same either way?  The `dep` (not `peerDep`) could be a version range, but the resolution takes precedence anyway.  I think I needed this for the moment since the sup-packages aren't published on `npm` yet, so it broke when trying to resolve a version range.

Reply:
@varl technically, since there are no "deep" dependencies on `@dhis2/app-service-{config|data}` the `resolutions` were superfluous, so I've removed them but this is still required (at least until the services are published)

Reply:
Actually I think the `resolutions` are more important than the `dependencies`, we can remove the dependency entries once the service packages have been published.